### DISCUSSION TITLE
Loader for segments instead of segment

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1842] [Elements (button, image, label, loader, ...)] [answered]
If you add "loading" class to a segment you can have centered loading for that segment.
So, in my case I have multiple segments in a "segments" container.
Do you know any way to add loading to segments container to have one loader instead of multiple loading for each segment?

**Comment by lubber-de** [comment, marked as answer]
I prepared the PR #1844 to support that.
In the meantime you could try a workaround and add `form` to your segments node, so it becomes `ui loading form segments`
